src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -25,6 +32,7 @@ private List<Product> getAllProductsFromShoppingCart(Long userId) {
         Product xiaomi = new Product("Xiaomi 12", BigDecimal.valueOf(499));
         List<Product> products = List.of(iphone, macBook, xiaomi);
         // TODO: add log message about successful fetched data from DB
+        orderServiceImplLogger.info("Data was successfully fetched from DB");
```

Comment:
Please add information regarding fetched data. We most commonly check logs when there is a problem reported and we require as much info as possible to find, reproduce and fix bug asap